src/ngraph/runtime/backend_manager.cpp

```diff
@@ -32,12 +32,18 @@
 using namespace std;
 using namespace ngraph;
 
+#ifdef NGRAPH_STATIC_LIB_ENABLE
+#define DLERROR() ""
+#else
 #ifdef _WIN32
 #define CLOSE_LIBRARY(a) FreeLibrary(a)
 #define DLSYM(a, b) GetProcAddress(a, b)
+#define DLERROR() dlerror()
```

Comment:
This should be in the other branch of the `#if`.

Reply:
It's in all three branches; it's a NO-OP not on _WIN32.

Reply:
That's confusing. Didn't know there is `dlerror` function on Windows.

Reply:
Just to be clear: I'm talking about the implementation of the macro. `dlerror` sounds more posix-like.